go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	google.golang.org/grpc v1.42.0
 	k8s.io/apimachinery v0.24.3
 	k8s.io/klog/v2 v2.70.1
-	k8s.io/kubernetes v1.20.6
+	k8s.io/kubernetes v1.26.0
```

Comment:
you need to rerun "go mod tidy" & "go mod vendor" again after you modify the go.mod

I'd recommend you revert all your commits in this branch and do this again.

Reply:
Thanks for catching it.  Reverted and (hopefully) got it right this time.